tee-worker/identity/enclave-runtime/src/initialization/global_components.rs

```diff
@@ -106,6 +109,7 @@ pub type EnclaveGetter = Getter;
 pub type EnclaveTrustedCallSigned = TrustedCallSigned;
 pub type EnclaveStf = Stf<EnclaveTrustedCallSigned, EnclaveGetter, StfState, Runtime>;
 pub type EnclaveStateKeyRepository = KeyRepository<Aes, AesSeal>;
+pub type EnclaveIDGraphKeyRepository = KeyRepository<Aes256Key, Aes256Seal>;
```

Comment:
How about `EnclaveAccountStoreKeyRepository`?

Reply:
Yes I changed it locally - haven't pushed it yet, thanks for checking